### PR TITLE
Fixed Error Statements

### PR DESCRIPTION
**Pull request by shivabalachandran**
I have not defined the error message correctly in the previous commit. Fixing in this commit.

### CODE REVIEW

File: modules/integration/tests-integration/bpmn/src/test/java/org/wso2/bps/integration/tests/bpmn/DeployUnDeployBPMNPackageTestCase.java [hunk 1]
```diff
@@ -73,7 +73,7 @@ public void deployUnDeployBPMNPackage() throws Exception {
             Assert.fail("Package Not Present");
```
Comment (chathurace) [inline modules/integration/tests-integration/bpmn/src/test/java/org/wso2/bps/integration/tests/bpmn/DeployUnDeployBPMNPackageTestCase.java]: Assert.fail() message should be "Package still exists after undeployment"

Comment (shivabalachandran) [inline modules/integration/tests-integration/bpmn/src/test/java/org/wso2/bps/integration/tests/bpmn/DeployUnDeployBPMNPackageTestCase.java]: Fixed in all test cases. Please find the changes in the latest commit.

**Comment by ramith** [comment]
git rebase -i HEAD~2 might be useful.

File: modules/integration/tests-common/admin-clients/src/main/java/org/wso2/bps/integration/common/clients/bpmn/ActivitiRestClient.java [hunk 2]
```diff
@@ -63,7 +63,7 @@
     public ActivitiRestClient(String portM, String hostnameM) {
         port = Integer.parseInt(portM);
         hostname = hostnameM;
-        serviceURL = "http://" + hostnameM + ":" + portM + "/bpmnrest/";
+        serviceURL = "http://" + hostnameM + ":" + portM + "/bpmn/";
```
Comment (imesh) [inline modules/integration/tests-common/admin-clients/src/main/java/org/wso2/bps/integration/common/clients/bpmn/ActivitiRestClient.java]: Better to use java.net.URL for constructing URLs.

Comment (shivabalachandran) [inline modules/integration/tests-common/admin-clients/src/main/java/org/wso2/bps/integration/common/clients/bpmn/ActivitiRestClient.java]: Changed variable type from string to URL. Please PR https://github.com/wso2/product-bps/pull/43>

Thank you.

**Pull request merged**